README.md

```diff
@@ -19,7 +19,7 @@ repositories {
 
 dependencies {
     // Include the sdk as a dependency
-    implementation 'com.microsoft.graph:microsoft-graph-beta:0.6.0-SNAPSHOT'
+    implementation 'com.microsoft.graph:microsoft-graph-beta:0.7.0-SNAPSHOT'
```

Comment:
I assume we don't need anything special for V3?

Reply:
I'm not sure what the question is? do you mean other dependencies? beta should pull core which should pull the rest

Reply:
I mean, no special versioning for V3, we continue bumping minor.

Reply:
there are still two pending question regarding whether we allow v1 and beta to stand side by side in an application and whether this beta service library should be published as a non-snapshot. I figured that as long as those questions are not answered by PMs, I shouldn't go implement any fancy versioning to avoid confusion and potentially having to backtrack.
https://github.com/microsoftgraph/msgraph-beta-sdk-java/issues/39
https://github.com/microsoftgraph/msgraph-beta-sdk-java/issues/49

Reply:
which is OK because it is beta, but I just wanted to make sure. I remember for .NET beta, there was a discussion about changing the versioning.